src/views/Lottery/components/YourPrizesCard/index.tsx

```diff
@@ -7,21 +7,24 @@ import PrizesWonContent from './PrizesWonContent'
 import NoPrizesContent from './NoPrizesContent'
 
 const StyledCard = styled(Card)`
+  margin-top: 16px;
+
+  ${(props) => `
```

Comment:
Wrapping this in a function with props is not necessary. This is the pattern we use around the site.

```
${({ theme }) => theme.mediaQueries.sm} {

}
```

Reply:
done